### PR TITLE
use /hack/e2e.sh to run a single node kind cluster for develop

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

add `helm` and `kind` soft link to `/usr/local/bin`, then we can use `helm ls` directly in the container.

run a single master development env with:

```
$ ./hack/run-in-container.sh
$ SKIP_BUILD=y SKIP_IMAGE_BUILD=y CLUSTER=demo PROVIDER=kind SKIP_DOWN=y SKIP_GINKGO=y KUBE_VERSION=v1.16.4 KUBE_WORKERS=0 SKIP_IMAGE_LOAD=y ./hack/e2e.sh
```

### Check List 

Tests 

 - No code

Code changes

 - Has CI related scripts change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
use `/hack/e2e.sh` to run a single node kind cluster for develop
```

### CODE REVIEW

File: hack/lib.sh [hunk 1]
```diff
@@ -103,6 +103,7 @@ function hack::ensure_helm() {
     fi
     local HELM_URL=http://storage.googleapis.com/kubernetes-helm/helm-v${HELM_VERSION}-${OS}-${ARCH}.tar.gz
     curl --retry 10 -L -s "$HELM_URL" | tar --strip-components 1 -C $OUTPUT_BIN -zxf - ${OS}-${ARCH}/helm
+    ln -s $OUTPUT_BIN/helm /usr/local/bin/helm
```
Comment (cofyc) [inline hack/lib.sh]:
```suggestion
    if [ "${HOSTNAME:-}" == "tidb-operator-dev" ]; then
        ln -s $OUTPUT_BIN/helm /usr/local/bin/helm
    fi
```

File: hack/lib.sh [hunk 2]
```diff
@@ -123,6 +124,7 @@ function hack::ensure_kind() {
     curl --retry 10 -L -o $tmpfile https://github.com/kubernetes-sigs/kind/releases/download/v${KIND_VERSION}/kind-$(uname)-amd64
     mv $tmpfile $KIND_BIN
     chmod +x $KIND_BIN
+    ln -s $KIND_BIN /usr/local/bin/kind
```
Comment (cofyc) [inline hack/lib.sh]:
```suggestion
    if [ "${HOSTNAME:-}" == "tidb-operator-dev" ]; then
        ln -s $KIND_BIN /usr/local/bin/kind
    fi
```

**Comment by weekface** [comment]
@cofyc @Yisaer PTAL

File: hack/e2e.sh [hunk 3]
```diff
@@ -481,12 +485,22 @@ else
     exit 1
 fi
 
+if [ ! -f /usr/local/bin/helm ]; then
```
Comment (cofyc) [inline hack/e2e.sh]:
```suggestion
if [ "${HOSTNAME:-}" == "tidb-operator-dev" -a ! -f /usr/local/bin/helm ]; then
```

File: hack/e2e.sh [hunk 4]
```diff
@@ -481,12 +485,22 @@ else
...
+    ln -s $OUTPUT_BIN/helm /usr/local/bin/helm
+fi
+
+if [ ! -f /usr/local/bin/kind ]; then
```
Comment (cofyc) [inline hack/e2e.sh]:
```suggestion
if [ "${HOSTNAME:-}" == "tidb-operator-dev" -a ! -f /usr/local/bin/kind ]; then
```

**Pull request merged**